pkg/controller/plan/adapter/ovirt/builder.go

```diff
@@ -120,6 +120,35 @@ var osMap = map[string]string{
 	"windows_11":           "win11",
 }
 
+var clusterCpuMap = map[string]string{
+	"Intel Nehalem Family":                   "Nehalem",
+	"Secure Intel Nehalem Family":            "Nehalem,+spec-ctrl,+ssbd",
+	"Intel Westmere Family":                  "Westmere",
+	"Secure Intel Westmere Family":           "Westmere,+pcid,+spec-ctrl,+ssbd",
+	"Intel SandyBridge Family":               "SandyBridge",
+	"Secure Intel SandyBridge Family":        "SandyBridge,+pcid,+spec-ctrl,+ssbd,+md-clear",
+	"Intel IvyBridge Family":                 "IvyBridge",
+	"Secure Intel IvyBridge Family":          "IvyBridge,+pcid,+spec-ctrl,+ssbd,+md-clear",
+	"Intel Haswell Family":                   "Haswell-noTSX",
+	"Secure Intel Haswell Family":            "Haswell-noTSX,+spec-ctrl,+ssbd,+md-clear",
+	"Intel Broadwell Family":                 "Broadwell-noTSX",
+	"Secure Intel Broadwell Family":          "Broadwell-noTSX,+spec-ctrl,+ssbd,+md-clear",
+	"Intel Skylake Client Family":            "Skylake-Client",
+	"Secure Intel Skylake Client Family":     "Skylake-Client-noTSX-IBRS,+ssbd,+md-clear,-mpx",
+	"Intel Skylake Server Family":            "Skylake-Server,-hle,-rtm,-mpx",
+	"Secure Intel Skylake Server Family":     "Skylake-Server-noTSX-IBRS,+ssbd,+md-clear,-mpx",
+	"Intel Cascadelake Server Family":        "Cascadelake-Server,-hle,-rtm,-mpx",
+	"Secure Intel Cascadelake Server Family": "Cascadelake-Server-noTSX,-mpx",
+	"Intel Icelake Server Family":            "Icelake-Server-noTSX,-mpx",
+	"Secure Intel Icelake Server Family":     "Icelake-Server-noTSX,+arch-capabilities,+rdctl-no,+ibrs-all,+skip-l1dfl-vmentry,+mds-no,+pschange-mc-no,-mpx",
+	"AMD Opteron G4":                         "Opteron_G4",
+	"AMD Opteron G5":                         "Opteron_G5",
+	"AMD EPYC":                               "EPYC",
+	"Secure AMD EPYC":                        "EPYC,+ibpb,+virt-ssbd",
+	"IBM POWER8":                             "POWER8",
+	"IBM POWER9":                             "POWER9",
+}
```

Comment:
about this mapping - did you take it from the latest cluster version on upstream?

Reply:
it's from cluster level 4.5 and above: https://github.com/oVirt/ovirt-engine/blob/master/packaging/dbscripts/upgrade/pre_upgrade/0000_config.sql#L1261

Reply:
yeah, it can also be different on downstream..
did you try to get the cluster values from the system options endpoint (BackendSystemOptionResource)?

Reply:
ok, using the endpoint `api/options/ServerCPUList?version=4.5` in ovirt we can get the list of specific version (or without the param and get all of them). so it is possible to be based on ovirt and not on hard-coded map.

Reply:
cool so let's do that